axelrod/random_.py

```diff
@@ -63,10 +71,16 @@ def randrange(a: int, b: int) -> int:
     return a + int(r)
 
 
-def seed(seed_):
-    """Sets a seed"""
-    random.seed(seed_)
-    numpy.random.seed(seed_)
+def random_vector(size):
+    """Create a random vector of values in [0, 1] that sums to 1."""
```

Comment:
Is something like:

```
vector = np.random.random(size)
return vector / np.sum(vector)
```

perhaps more efficient because of numpy magic (of course it could be the opposite) and more readable?

Reply:
Yeah that's fine. I don't think I originally wrote this function, it looks odd to me as well.